### PR TITLE
add support for initializing to the current working directory (#25)

### PR DESCRIPTION
**Pull request by phildow**
https://github.com/doc-ai/aleph/issues/25

Create a new directory _myproj_ in the current working directory and bootstrap there:

```
aleph init myproj
```

Bootstrap the project in the current working directory

```
aleph init .
```

Updated unit tests to check both paths.

### CODE REVIEW

File: aleph/cli/init/test_generator.py [hunk 1]
```diff
@@ -21,11 +21,19 @@ def tearDown(self):
...
 
-    def test_generate_1(self):
+    def test_generate_run_with_new_project_name(self):
         args = types.SimpleNamespace(name='test_project')
         generator.run(args)
         project_dir = os.path.join(self.work_dir, args.name)
-        
+        self.check_that_project_is_correctly_bootstrapped(project_dir)
+
+    def test_generate_run_with_current_working_directory(self):
```
Comment (nkashy1) [inline aleph/cli/init/test_generator.py]: @phildow : Could you add a test that demonstrates that `aleph init` fails if it is provided a non `.` path which already exists?

File: aleph/cli/init/generator.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 from aleph.cli.utils.exceptions import GeneratorException
 
+INIT_TO_CWD = '.'
```
Comment (nkashy1) [inline aleph/cli/init/generator.py]: Also allow `./`?

Comment (phildow) [reply aleph/cli/init/generator.py]: Yeah, I think that's a good idea

File: aleph/cli/init/test_generator.py (hunk 1 above)
Comment (phildow) [reply aleph/cli/init/test_generator.py]: Can do

**Pull request merged**